scripts/BI-115.ts

```diff
@@ -0,0 +1,47 @@
+
+import { Agenda } from '../modules/turnos/schemas/agenda';
+import { Prestacion } from '../modules/rup/schemas/prestacion';
+import { Profesional } from '../core/tm/schemas/profesional';
+import moment = require('moment');
+import { getTurno } from 'modules/turnos/controller/agenda';
```

Comment:
```suggestion
import { getTurno } from '../modules/turnos/controller/agenda';
```
Me dice que no lo encuentra con la ruta absoluta